README.md

```diff
@@ -1,3 +1,13 @@
 # counties-demo [![Build Status](https://travis-ci.org/probcomp/counties-demo.svg?branch=master)](https://travis-ci.org/probcomp/counties-demo)
 
-A demonstration of how to use `probcomp/bayesrest`. Someday.
+## What the counties-demo repo is
+
+The `counties-demo` repo contains a Jupyter notebook that demonstrates the capabilities of the Bayesian Database Search API.  We have loaded the API with a tabular dataset of over 3,000 counties (rows) and about 130 demographic, income, drug use, and political statistics (columns) of each of those counties.  And we demonstrate the API with this Jupyter notebook by calling three of the API's endpoints.  Further documentation on the API can be found here[www.tbd.com].
+
+## Getting Started
```

Comment:
We should probably mention [Git LFS](https://git-lfs.github.com/) somewhere in here.